2_Operations/work-process.md

```diff
@@ -1,65 +1,52 @@
-# Work process
-
-## Scrum process
-> *Main document: [Scrum](scrum.md)*
-
-Our work process is based on [Scrum](https://en.wikipedia.org/wiki/Scrum_(software_development)), but modified to suit to our specific needs:
-
- * We are our own *product owners*, as we have no customer projects.
- * We work asynchronously, so keep scheduled meetings to the bare minimum.
- * We schedule our work into two week long *sprints*.
- * We manage our sprints with GitHub and ZenHub.
 
+# Work Process
 
 ## Schedule
 
- * Every weekday morning we hold a [daily standup](standup.md) meeting.
+ * Every weekday morning we hold a daily standup meeting.
 
 Sprint meetings replace the daily standup on these days:
 
- * A Wednesday is the start of a sprint with [sprint planning](scrum.md#Sprint_Planning).
- * The Tuesday, two weeks later, marks the end of the sprint with a [sprint retrospective](scrum.md#Sprint_Retrospective).
+ * A Wednesday is the start of a sprint with sprint planning.
+ * The Tuesday, two weeks later, marks the end of the sprint with a sprint retrospective.
  * Then Wednesday, the next day, is usually the start of the next sprint.
 
+The last Monday morning of the sprint everyone should open up the Scrum board and ask themselves: "How can I help close whatever is still opened?". Repeat the same after lunch and on Tuesday morning.
 
-## Creating issues and user stories
-> *Main document: [Guide for user stories](user-stories.md)*
+For each new sprint, the Scrum master will create an operations sprint release that is used to store the user story screencast demos. The template for this release is copied from [RELEASE_TEMPLATE.md](https://github.com/niteoweb/operations/raw/master/.github/RELEASE_TEMPLATE.md) in the operations repo.
```

Comment:
Do we have demos from both ebn and woocart sprints in the same release? If not, the releases should be in their respective repositories.

Reply:
In their respective repositories.